docs/reference/changelog.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 *********
 
+0.7.1 (2025-MM-DD)
+
+Bug Fixes
+=========
+
+- The "bare" base for Rockcraft was not properly handled. An error will now be raised
+  if "bare" is specified with no "build-base".
```

Comment:
```suggestion
- The bare base for Rockcraft wasn't properly handled. An error will now be raised
  if ``bare`` is specified with no ``build-base``.
```

I'd prefer if we avoid airquotes because of its impact on tone.